src/PresentationalComponents/Inventory/helpers.js

```diff
@@ -13,6 +13,7 @@ export const paginatedRequestHelper = async ({
   SID,
   pathway,
   rule,
+  selectedTags,
 }) => {
   let options = {
```

Comment:
There are some issues to get around when doing this. I'd like to leave this PR as is since its currently broken, and include the refactor for the options in the advisor refactor PR.

Reply:
I have it included in the refactor now @bastilian